File: Códigos/Ouve Fácil-SRC/class/status.php
<?php

class status{
    private $codStatus;
    private $nome;

    public function setCodStatus($codstatus){
        $this->codStatus = $codStatus;
    }
    public function getCodStatus(){
        return $this->codStatus;
    }
    public function setNome($nome){
        $this->nome = $nome;
    }
    public function getNome(){
        return $this->nome;
    }
}
?>

File: Códigos/OuveFacil/OuveFacil/app/src/main/java/domain/model/Status.java
package domain.model;

public class Status {

    private int codStatus;
    private String nome;

    public int getCodStatus() {
        return codStatus;
    }

    public void setCodStatus(int codStatus) {
        this.codStatus = codStatus;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    @Override
    public String toString() {
        return this.nome;
    }
}
